src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -8,6 +8,14 @@ description: "Learn more about Sentry’s integration platform and how it provid
 
 Sentry’s integration platform provides a way for external services to interact with Sentry using [webhooks](/product/integrations/integration-platform/webhooks/), [UI components](/product/integrations/integration-platform/ui-components/), and the [REST API](/api). Integrations using this platform are first-class actors within Sentry.
 
+## Example App
+
+If you're new to the integration platform, it might help to get started with an example project. To help you out, we've built a [example app](https://github.com/getsentry/integration-platform-example), which you can use as a starting point. It demonstrates the different features and data available for your application:
```

Comment:
```suggestion
If you're new to the integration platform, it might help to get started with an example project. To help you out, we've built an [example application](https://github.com/getsentry/integration-platform-example), which you can use as a starting point. It demonstrates the different features and data available for your application:
```